POEApi.Model/POEModel.cs

```diff
@@ -300,6 +300,11 @@ private void getImageWithEvents(Item item)
             try
             {
                 getImageWithEvents(GetItemName(item), item.IconURL);
+
+                if (item.ItemType == ItemType.DivinationCard)

```

Comment:
it will do it twice for divcards which is inefficient

if (Divcard)
   get()
else
   get()

Reply:
the first call will get the icon for the stash and the second the actual image for the popup